src/MessagingAttachment/MessagingAttachment.tsx

```diff
@@ -11,8 +11,17 @@ function cssClass(element: string) {
   return `MessagingAttachment--${element}`;
 }
 
-type Props = {
+type AttachmentPreviewProps = {
+  // TODO: should any of these be optional vs required (different then they currently are)?
   attachmentURL: string;
+  closeButtonAriaLabel?: string;
+  downloadButtonTextDeskop?: string;
+  downloadButtonTextMobile?: string;
+};
+
+type Props = {
+  // TODO: if any of the above props are required, this should be required. Else it could be optional?
```

Comment:
`attachmentURL` is required (right?), so this is indeed required as well.

Reply:
Yes it is, I wasn't sure if it needed to be, but I think it does...